src/app/component/ImageUpload/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useRef, useState } from 'react';
```

Comment:
This one is for the team: while using index.js inside a named dir is standard, I believe it adds more issues. For example, in my IDE, all the files would be named index.js. I would need to hover on the filename to get the dir name. This is specially annoying when you need to search up a file name, but can't since every file is named index.js 

But again, this is for the whole team to decide.

Reply:
Yeah, agree. I think we will likely keep index.js file only for import and export in the future